tests/suites/test_suite_mps.function

```diff
@@ -834,10 +841,6 @@ void mbedtls_mps_reader_random_usage( int num_out_chunks,
         ASSERT_ALLOC( acc, acc_size );
     }
 
-    /* This probably needs to be changed because we want
```

Comment:
Should this be replaced by calling `mbedtls_test_rnd_pseudo_reset_state()`?

Reply:
`dispatch_test` will reset the global rng state used by `mbedtls_test_rnd_pseudo_rand`.